src/components/TouchBar/Markers/Markers.jsx

```diff
@@ -66,14 +65,42 @@ class Markers extends Component {
     return markers;
   }
 
-  // Determines whether this marker is visible by checking occlusion between
-  // all nodes that are currently visible on the screen
+  //Determines size of the marker
+  static determineSize(nodeRadius){
+
+    let size = nodeRadius * 0.1;

```

Comment:
Suggest creating a const with the 0.1 value, with a variable name that explains the value.